src/main/java/ch/ivyteam/ivy/maven/engine/EngineControl.java

```diff
@@ -198,13 +200,41 @@ private void findStartEngineUrl(String newLine)
     if (newLine.contains("info page of Axon.ivy Engine") && !engineStarted.get())
     {
       String url = StringUtils.substringBetween(newLine, "http://", "/");
-      url = "http://" + url + "/ivy/";
+      url = "http://" + url + "/";
+      url += evaluateDefaultContext(url);
       context.log.info("Axon.ivy Engine runs on : " + url);
       context.properties.setMavenProperty(Property.TEST_ENGINE_URL, url);
       engineStarted.set(true);
     }
   }
 
+  private String evaluateDefaultContext(String url)
+  {
+    HttpURLConnection connection = null;
+    try
+    {
+      connection = (HttpURLConnection) new URL(url).openConnection();
+      connection.setRequestMethod("GET");
+      connection.setInstanceFollowRedirects(false);
+      connection.getResponseCode();
+      String location = connection.getHeaderField("Location");
+      String defaultContext = StringUtils.removeEnd(location, "/sys/info.xhtml");
```

Comment:
looks prone to errors when we adjust any URI....
I'd use something a little more resilient... like cutting 'known start url from location' ... and use as context the thing between /sys and the well known prefix...

Reply:
the problem is here there is no start. The location is:
LTS: ivy/sys/info.xhtml
LE: /sys/info.xhtml

Or do you mean to make a substring before '/sys'?